README.md

```diff
@@ -140,12 +145,15 @@ python setup.py develop
 
 ## GPU acceleration
 
-The available frontends are PyTorch (provided with the backends `torch`, `torch_skcuda`), TensorFlow (with the backend
-`tensorflow`), and NumPy (with the backend `numpy`).
+Certain frontends, `numpy` and `sklearn`, only allow processing on the CPU and are therefore slower. The `torch`, `tensorflow`, and `keras` frontends, however, also support GPU processing, which can significantly accelerate computations. Additionally, the `torch` backend supports an optimized `skcuda` backend which currently provides the fastest performance in computing scattering transforms. In 2D, it may be instantiated using:
```

Comment:
"The `torch`, `tensorflow`, and `keras` frontends also support GPU processing, which can significantly accelerate computations." ?

Reply:
Is there a problem with this sentence or was that a suggestion to fix? It looks like the sentence in the doc is exactly that.

Reply:
Suggestion, theres a however in the current document which kinda ruins my reading experience.